Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,77 @@
+﻿using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    [SerializeField] private float speed;
+    [SerializeField] private float jumpForce;
+    [SerializeField] private GameObject body;
+
+    private IUnityService unityService;
+
+    private Movement movement;
+    private Animator animator;
+    private SpriteRenderer bodySpriteRenderer;
+    private Rigidbody2D rb;
+
+    private bool isJumping;
+
+    private void Start()
+    {
+        movement = new Movement(speed, jumpForce);
+        animator = GetComponent<Animator>();
+        rb = GetComponent<Rigidbody2D>();
+
+        if (unityService == null)
+        {
+            unityService = new UnityService();
+        }
+
+        ObtainRandomColor();
+    }
+
+    private void FixedUpdate()
+    {
+        var horizontalSpeed = movement.GetHorizontalSpeed();
+
+        animator.SetFloat("HorizontalSpeed", horizontalSpeed);
+
+        if (horizontalSpeed > 0) // maybe more optimised than adding Vector3(0,0,0) to position even when standing still

```

Comment:
Yeah, may be. To be honest I don't know if it is more optimal or not, worth checking.

One small thing that might be worth changing is making the reason of the condition more understandable by moving it to separate method:
```suggestion
        if (shouldMove()) // maybe more optimised than adding Vector3(0,0,0) to position even when standing still
```